beets/autotag/hooks.py

```diff
@@ -297,7 +297,7 @@ def __init__(self):
         self._penalties = {}
 
     @LazyClassProperty
-    def _weights(cls):
+    def _weights(self):
```

Comment:
Hmm; this one is actually a class method—the style checker just doesn't know that!